app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailFragment.java

```diff
@@ -145,7 +130,6 @@ public void onRefreshComplete() {
 
     @Override
     public void showEvent(Event event) {
-        listener.onEventLoaded(event, false);
```

Comment:
this was responsible for setting drawer header values (event name, time) when the app is restarted with stored event id. (event key in prefs not equal to -1).
please add this functionality accordingly